### PR TITLE
Update draft-boucadair-nmop-rfc3535-20years-later.md

### PR DESCRIPTION
**Pull request by luismcontreras**
Added as new observation the network APIfication (e.g., peering API).

### CODE REVIEW

File: draft-boucadair-nmop-rfc3535-20years-later.md [hunk 1]
```diff
@@ -308,6 +308,10 @@ That may slow down the protocol specification, though.
 While there are open-source implementations for NETCONF (e.g., NETOPEER), the gRPC/gNMI suite seems to have more support for tools on the client side.
 For example, "ygot" generates structures from YANG models and these can easily be used by a client to configure a device with gNMI. NETCONF is not supported though (we need the XML tags).
 
+## Network APIfication
+
+APIs are getting momentum as means of interworking between parties, also at the time of providing network services. An example of that is {{?I-D.ramseyer-grow-peering-api}} for the purpose of dynamically establishing BGP peering sessions between Autonomous Systems of different administrative domains. Thus, the definition of APIs, independent of the realization of the supportive model, could be generalize for fast development, prototyping, and validation.
```
Comment (boucadair) [inline draft-boucadair-nmop-rfc3535-20years-later.md:313]:
```suggestion
APIs are getting momentum as means of interworking between parties, also at the time of providing network services. As an example, {{?I-D.ramseyer-grow-peering-api}} defines an API for dynamically establishing BGP peering sessions between Autonomous Systems of different administrative domains. That same objective is also covered by the YANG data model defined in {{?I-D.ietf-opsawg-teas-attachment-circuit}} as exemplified in Appendix A.10. Tools such as YANG/OpenAPI transforms are key to leverage existing data models and allow for better integration and mapping to actual realization models.

Readily available API specifications could be generalized for fast development, prototyping, and validation. 
```

**Review by boucadair — CHANGES_REQUESTED** [review]
Thanks. 

Building on a data model is more future proof than using APIs verbs, etc and independent of how the request will be handled internally. We need tools to such as https://github.com/corenova/yang-swagger

File: draft-boucadair-nmop-rfc3535-20years-later.md (hunk 1 above)
Comment (luismcontreras) [reply draft-boucadair-nmop-rfc3535-20years-later.md]: Ok!

File: draft-boucadair-nmop-rfc3535-20years-later.md [hunk 2]
```diff
@@ -308,17 +308,21 @@ That may slow down the protocol specification, though.
...
```
Comment (boucadair) [inline draft-boucadair-nmop-rfc3535-20years-later.md:313]:
```suggestion
APIs are getting momentum as means of interworking between parties, also at the time of providing network services. As an example, {{?I-D.ramseyer-grow-peering-api}} defines an API for dynamically establishing BGP peering sessions between Autonomous Systems of different administrative domains. That same objective is also covered by the YANG data model defined in {{?I-D.ietf-opsawg-teas-attachment-circuit}} as exemplified in Appendix A.10. Tools such as YANG/OpenAPI transforms are key to leverage existing data models and allow for better integration and mapping to actual realization models.

Readily available API specifications could be generalized for fast development, prototyping, and validation.
```

**Pull request merged**